lm75bd/lm75bd.c

```diff
@@ -27,6 +28,29 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
+
+  uint8_t temp_pointer_byte = 0x00U; // selects temp register at 00 LSB
+  uint8_t *send_buf = &temp_pointer_byte;
+
+  i2cSendTo(devAddr, send_buf, 1);
```

Comment:
Use sizeof(send_buf) instead of 1. Also we use camelCase for variables on the team